<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>569</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Музичний каталог</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="addPassButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>371</width>
      <height>61</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: ;
    border: 2px solid #0511ff;
    color: #000000;
    padding: 12px 28px;
    text-align: center;
    text-decoration: none;
    display: inline-block;
    font-size: 18px;
    margin: 6px 3px;
    cursor: pointer;
    border-radius: 20px;
}
QPushButton:hover {
    background-color: #03fbff;
    color: white;
}
</string>
    </property>
    <property name="text">
     <string>Додати композицію</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="addMusicTitleLineEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>361</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    background-color: transparent;
    font-size: 18px;
    border: 2px solid #0511ff;
    color: #000000;
    padding: 10px;
    border-radius: 10px;
    border: 1px solid #0511ff;
}

QLineEdit:focus {
    border-color: #0511ff;
    border-width: 3px;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Виконавець</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="addMusicValueLineEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>361</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    background-color: transparent;
    font-size: 18px;
    border: 2px solid #FF6347;
    color: #000000;
    padding: 10px;
    border-radius: 10px;
    border: 1px solid #0511ff;
}

QLineEdit:focus {
    border-color: #0511ff;
    border-width: 3px;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Назва пісні</string>
    </property>
   </widget>
   <widget class="QPushButton" name="editPassButton">
    <property name="geometry">
     <rect>
      <x>418</x>
      <y>60</y>
      <width>351</width>
      <height>61</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: transparent;
    border: 2px solid #0511ff;
    color: #000000;
    padding: 12px 28px;
    text-align: center;
    text-decoration: none;
    display: inline-block;
    font-size: 18px;
    margin: 6px 3px;
    cursor: pointer;
    border-radius: 20px;
}
QPushButton:hover {
    background-color: #03fbff;
    color: white;
}
</string>
    </property>
    <property name="text">
     <string>Редагувати композицію</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editMusicTitleLineEdit">
    <property name="geometry">
     <rect>
      <x>428</x>
      <y>130</y>
      <width>341</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    background-color: transparent;
    font-size: 18px;
    border: 2px solid #FF6347;
    color: #000000;
    padding: 10px;
    border-radius: 10px;
    border: 1px solid #0511ff;
}

QLineEdit:focus {
    border-color: #0511ff;
    border-width: 3px;
}
</string>
    </property>
    <property name="placeholderText">
     <string>Веедіть виконавця</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editMusicValueLineEdit">
    <property name="geometry">
     <rect>
      <x>428</x>
      <y>190</y>
      <width>341</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    background-color: transparent;
    font-size: 18px;
    border: 2px solid #FF6347;
    color: #000000;
    padding: 10px;
    border-radius: 10px;
    border: 1px solid #0511ff;
}

QLineEdit:focus {
    border-color: #0511ff;
    border-width: 3px;
}
</string>
    </property>
    <property name="placeholderText">
     <string>Введіть назву пісні</string>
    </property>
   </widget>
   <widget class="QListWidget" name="musicListWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>751</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QListWidget {
    background-color: transparent;
    border: 2px solid #FF6347;
    color: black;
    padding: 10px;
    border-radius: 10px;
    border: 1px solid #0511ff;
}

QListWidget::item {
    padding: 5px;
}

QListWidget::item:selected {
    background-color: #03fbff;
    color: black;
}
</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deletePassButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>251</width>
      <height>61</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="0">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="0">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="0">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="PlaceholderText">
        <brush brushstyle="SolidPattern">
         <color alpha="128">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="0">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="0">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="0">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="PlaceholderText">
        <brush brushstyle="SolidPattern">
         <color alpha="128">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="0">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="0">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="0">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="PlaceholderText">
        <brush brushstyle="SolidPattern">
         <color alpha="128">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: transparent;
    border: 2px solid #0511ff;
    color: #000000;
    padding: 12px 28px;
    text-align: center;
    text-decoration: none;
    display: inline-block;
    font-size: 18px;
    margin: 6px 3px;
    cursor: pointer;
    border-radius: 20px;
}
QPushButton:hover {
    background-color: #03fbff;
    color: white;
}
</string>
    </property>
    <property name="text">
     <string>Видалити композицію</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveToFileButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>240</y>
      <width>251</width>
      <height>61</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: transparent;
    border: 2px solid #0511ff;
    color: #000000;
    padding: 12px 28px;
    text-align: center;
    text-decoration: none;
    display: inline-block;
    font-size: 18px;
    margin: 6px 3px;
    cursor: pointer;
    border-radius: 20px;
}
QPushButton:hover {
    background-color: #03fbff;
    color: white;
}
</string>
    </property>
    <property name="text">
     <string>Зберегти у файл</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadFromFileButton">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>240</y>
      <width>251</width>
      <height>61</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: transparent;
    border: 2px solid #0511ff;
    color: #000000;
    padding: 12px 28px;
    text-align: center;
    text-decoration: none;
    display: inline-block;
    font-size: 18px;
    margin: 6px 3px;
    cursor: pointer;
    border-radius: 20px;
}
QPushButton:hover {
    background-color: #03fbff;
    color: white;
}
</string>
    </property>
    <property name="text">
     <string>Зчитати з файлу</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
